client/www/scripts/modules/common/lb-build.js

```diff
@@ -424,43 +424,6 @@ module.factory(
 
         /**
          * @ngdoc method
-         * @name BuildDeployAPI.Build#createChangeStream
-         * @methodOf BuildDeployAPI.Build
-         *
-         * @description
-         *
-         * Create a change stream.
-         *
-         * @param {Object=} parameters Request parameters.
-         *
-         *   This method does not accept any parameters.
-         *   Supply an empty object or omit this argument altogether.
-         *
-         * @param {Object} postData Request data.
-         *
-         *  - `options` – `{object=}` - 
-         *
-         * @param {function(Object,Object)=} successCb
-         *   Success callback with two arguments: `value`, `responseHeaders`.
-         *
-         * @param {function(Object)=} errorCb Error callback with one argument:
-         *   `httpResponse`.
-         *
-         * @returns {Object} An empty reference that will be
-         *   populated with the actual data once the response is returned
-         *   from the server.
-         *
-         * Data properties:
-         *
-         *  - `changes` – `{ReadableStream=}` - 
-         */
-        "createChangeStream": {
-          url: urlBase + "/Builds/change-stream",
-          method: "POST"
-        },
```

Comment:
I think this file should not be changed.  I updated it last week.  you might need to rebase before merging